### DISCUSSION TITLE
How to use pcmci/pcmciplus for time-series that come from separately collected time windows?

### DISCUSSION THREAD

**Discussion by f-arab** [jakobrunge/tigramite#261] [Q&A] [answered]
Hello,

We want to run PCMCI method on time-series that come from separately collected time windows instead of having one continuously recorded time-series. Does anyone know a quick way of implementing this on current PCMCI or PCMCIplus algorithms?

Thanks

**Comment by jakobrunge** [comment, marked as answer]
This can now be achieved with the multidata functionality: https://github.com/jakobrunge/tigramite/blob/master/tutorials/dataset_challenges/tigramite_tutorial_multiple_datasets.ipynb

Also see the J-PCMCI method that can overcome latent confounding using contexts: https://github.com/jakobrunge/tigramite/blob/master/tutorials/causal_discovery/tigramite_tutorial_jpcmciplus.ipynb
